cmd/osbuild-composer/main.go

```diff
@@ -70,6 +71,17 @@ func main() {
 		logrus.AddHook(hook)
 	}
 
+	if config.GlitchTipDSN != "" {
+		err = sentry.Init(sentry.ClientOptions{
+			Dsn: config.GlitchTipDSN,
+		})
+		if err != nil {
+			panic(err)
+		}
+	} else {
+		logrus.Warn("Sentry/Glitchtip was not initialized")
```

Comment:
Maybe something like "GLITCHTIP_DSN not configured, skipping initializing Sentry/Glitchtip"? Your variant sounds like composer attempted to initialize it, but wasn't successful in doing so.